src/utils/diff/parsers/common.ts

```diff
@@ -60,27 +60,48 @@ export abstract class BaseParser {
     abstract identity: string
     matchClientName = true
     clientNames: string[]
+
     abstract variableMethodPattern: RegExp
-    variableMethodKeywordPattern : RegExp | null = null
-    abstract variableNameCapturePattern: RegExp
-    abstract defaultValueCapturePattern: RegExp
+    orderedParameterPatterns: RegExp[] | null = null
+    namedParameterDelimiter = '='
+    namedParameterPatternMap: Record<string, RegExp> | null = null
+
     commentCharacters: string[] = ['//']
 
     constructor(extension: string, protected options: ParseOptions) {
         this.clientNames = [...(options.clientNames || []), 'dvcClient']
     }
 
     buildRegexPattern(): RegExp {
-        const variableRegex = this.variableMethodKeywordPattern ?
-            `(?:${this.variableMethodKeywordPattern.source}|${this.variableMethodPattern.source})` :
-            this.variableMethodPattern.source
-        const clientNamePattern = this.matchClientName ? new RegExp(`(?:${this.clientNames.join('|')})`).source : ''
+        const clientNamePattern = this.matchClientName
+            ? new RegExp(`(?:${this.clientNames.join('|')})`).source
+            : ''
+        const orderedParameters = this.orderedParameterPatterns
+            ? this.orderedParameterPatterns
+                .map((p) => p.source)
+                .join(new RegExp('\\s*,\\s*').source)
```

Comment:
you could use a Regex literal here to avoid double escaping the backslashes 
`/\s*,\s*/`